helm/draughtsman-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: draughtsman
+  namespace: draughtsman
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmap
+      - secrets
```

Comment:
I think we can limit cm and secret to a single named one, as it's always the same

Reply:
yes it has sense